pkg/common/helper_test.go

```diff
@@ -0,0 +1,167 @@
+package common
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/suite"
+)
+
+type IsFileTestSuite struct {
+	suite.Suite
+	tempDir string
+}
+
+type IsDirTestSuite struct {
+	suite.Suite
+	tempDir string
+}
+
+type FileExistTestSuite struct {
+	suite.Suite
+	tempDir string
+}
+
+type StringSliceToIntSliceTestSuite struct {
+	suite.Suite
+}
+
+type RetryUntilSuccessfulTestSuite struct {
+	suite.Suite
+}
+
+func (suite *IsFileTestSuite) SetupTest()  {
+	var err error
+
+	// create a temp directory
+	suite.tempDir, err = ioutil.TempDir("", "isfile-test")
+	suite.Require().NoError(err)
+}
+
+func (suite *IsFileTestSuite) TestIsFileFuncPositive() {
+	//create temp file
+	tempFile, err := ioutil.TempFile(suite.tempDir, "temp_file")
+	suite.Require().NoError(err)
+	suite.Assert().True(IsFile(tempFile.Name()))
```

Comment:
Don't use `Assert` - use `Require`. From the documentation:
```
The require package provides same global functions as the assert package, but instead of returning a boolean result they terminate current test.
```

This repeats itself in this PR.